frontend/src/components/SuperAdminPanel/EditOrganizations.tsx

```diff
@@ -96,19 +99,20 @@ const EditOrganizations = () => {
     loading,
     refresh: refreshOrganizations,
   } = useApiGet({
-    getFn: getOrganizations,
+    getFn: getAllOrganizations,
   })
```

Comment:
Bruk tanstack sin useQuery i stedet for useApiGet

Reply:
Hvilke spesifikke endringer må jeg gjøre i filen for å bruke useQuery?